crates/router/src/connector/payone.rs

```diff
@@ -559,3 +375,47 @@ impl api::IncomingWebhook for Payone {
         Err(report!(errors::ConnectorError::WebhooksNotImplemented))
     }
 }
+
+impl ConnectorErrorTypeMapping for Payone {
+    fn get_connector_error_type(
+        &self,
+        error_code: String,
+        _error_message: String,
+    ) -> ConnectorErrorType {
+        match error_code.as_str() {
+            "30001101" => ConnectorErrorType::BusinessError,
+            "30001100" => ConnectorErrorType::BusinessError,
+            "30001102" => ConnectorErrorType::BusinessError,
+            "30001104" => ConnectorErrorType::BusinessError,
+            "30001105" => ConnectorErrorType::BusinessError,
```

Comment:
You can group multiple `match` arms together:

```suggestion
            "30001101"
            | "30001100"
            | "30001102"
            | "30001104"
            | "30001105" => ConnectorErrorType::BusinessError,
```

Optional change, by the way.

Reply:
Bro, He said that keep them in order